wordpress-mobile/WordPress-iOS
# Dark mode: fixes for stats: maps and view more

Refs #12320

Fixes two problems in stats: the map colors, and the background color for "view more" rows.

![image](https://user-images.githubusercontent.com/517257/63305879-66b06100-c29d-11e9-9477-a66af7591cb4.png)


To test:
- open stats in dark mode
- ensure view more rows have the proper dark background
- ensure the map looks about right

### Note:
The map doesn't entirely transition dynamically when changed between light and dark mode. I intend to leave that for a future improvement.

Update release notes:
